### ISSUE TITLE
"efivars support missing: /sys/firmware/efi/efivars" on a BIOS computer

### ISSUE THREAD

**Issue by joakim-tjernlund** [fwupd/fwupd#2237]
**Describe the bug**
Multiple warnings on a BIOS only machine:

```
FuPluginUefi         kernel efivars support missing: /sys/firmware/efi/efivars
```
**Steps to Reproduce**
Steps to reproduce the behavior.

fwupdtool install <some fw cab>

**fwupd version information**
Please provide the version of the daemon and client.
```shell
$ fwupdmgr --version
```
client version:	1.4.4
compile-time dependency versions
	gusb:	0.3.4
	efivar:	37
daemon version:	1.4.4

**Additional questions**
- Operating system and version: Gentoo
- Have you tried rebooting? No
- Is this a regression? Not sure

**Comment by hughsie** [comment]
>  Multiple warnings on a BIOS only machine:
> FuPluginUefi         kernel efivars support missing: /sys/firmware/efi/efivars

What were you expecting?

**Comment by joakim-tjernlund** [comment]
> > Multiple warnings on a BIOS only machine:
> > FuPluginUefi         kernel efivars support missing: /sys/firmware/efi/efivars
> 
> What were you expecting?

I guess I should have provided more context ...
As I was upgrading an USB device which has nothing to do with UEFI, I figured I should not have to see these.
See https://github.com/fwupd/jabra-firmware/pull/3

**Comment by superm1** [comment]
Note; this is fixed in 1.5.0+, it is not planned to be backported to 1.4.x right now since 1.4.x actually makes a dummy device to tell you to switch modes from the uefi plugin, 1.5.x has a BIOS plugin for that.

**Comment by joakim-tjernlund** [comment]
> Note; this is fixed in 1.5.0+, it is not planned to be backported to 1.4.x right now since 1.4.x actually makes a dummy device to tell you to switch modes from the uefi plugin, 1.5.x has a BIOS plugin for that.

Sure, I am happy with that. Thanks.
